internal/exec/describe_workflows.go

```diff
@@ -36,6 +37,11 @@ func NewDescribeWorkflowsExec() DescribeWorkflowsExec {
 
 // ExecuteDescribeWorkflowsCmd executes `atmos describe workflows` CLI command.
 func (d *describeWorkflowsExec) Execute(atmosConfig *schema.AtmosConfiguration, describeWorkflowsArgs *DescribeWorkflowsArgs) error {
+	// Set the pager value from args
+	if describeWorkflowsArgs.Pager != "" {
```

Comment:
Let us not add the param here. If it is in atmos config. We should read from atmos config